heka/_service.sls

```diff
@@ -2,7 +2,7 @@
 
 {%- set server = salt['pillar.get']('heka:'+service_name) %}
 
-{%- if server.enabled %}
+{%- if server is defined %}
```

Comment:
in init.sls we use defined and in service.sls we use enabled because with this you have second tree where you can uninstall/remove this service because if you have only defined than if you remove this definition from your model, this service will will be present forever, 

it's just a standard not a rule